ci/testkomodo.sh

```diff
@@ -0,0 +1,13 @@
+
+copy_test_files () {
+    cp -r $CI_SOURCE_ROOT/tests $CI_TEST_ROOT/tests
+    cp $CI_SOURCE_ROOT/pyproject.toml $CI_TEST_ROOT
+}
+
+install_test_dependencies () {
+    pip install -e .
+}
+
+start_tests () {
+    pytest -n auto
```

Comment:
I don't think we want parallel tests here, our github action runners are weak and already loaded while this is running.